File: dashboard/src/app/shared/models/listing.ts
export interface IListing {
    Company: string;
    Series: string;
    SAS: string;
    Symbol: string;
}

export class Listing implements IListing { 
  
    public Company: string;
    public Series:string;
    public SAS:string;
    public Symbol:string;

    constructor(data: Partial<Listing>){
        Object.assign(this, data);
    }
  }

File: golang/pkg/contracts/listing.go
package contracts

// Listing
// swagger:model Listing
type Listing struct {
	Name      string `json:"Company" description:"Company name"`
	Symbol    string `json:"Symbol"  description:"Listing symbol"`
	SASSymbol string `json:"SAS"     description:"SAS symbol"`
	Series    string `json:"Series"  description:"Listing series"`
}
